library/chemistry/src/Tests.qs

```diff
@@ -1,14 +1,10 @@
-import Std.StatePreparation.ApproximatelyPreparePureStateCP;
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-import JordanWigner.JordanWignerClusterOperatorEvolutionSet.JordanWignerClusterOperatorPQRSTermSigns;
-import JordanWigner.OptimizedBEOperator.OptimizedBEXY;
-import JordanWigner.OptimizedBEOperator.SelectZ;
-import JordanWigner.StatePreparation.PrepareSparseMultiConfigurationalState;
-import JordanWigner.StatePreparation.PrepareUnitaryCoupledClusterState;
-import JordanWigner.Utils.JordanWignerInputState;
+
```

Comment:
It looks like maybe an extra line here.

Reply:
Not sure what you mean, but I verified consistent spacing in this file.
I did put an extra line between std lib imports and imports from this chemistry library, I think it makes it easier to understand - separation of stuff imported from elsewhere and stuff imported from neighboring files in the library.